LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Is there an advantage of using `LICENSE` over `LICENSE.md`?

Reply:
1. The `LICENSE` and `LICENSE.txt` are standard names for the license files recognized by different tools by default.
2. The Apache license provides explicit recommendations on how to apply the license text.
3. Because it's a well-known license (unlike custom EULA), there is no need to include the license text directly to the package distribution to reduce distribution metadata bloat. Because it's specified in the package metadata as `license = {file = "LICENSE.md"}` the whole text license is included into the distribution package metadata file. Changing it to `license = {text = "Apache-2.0"}` still includes the `LICENSE` file into the distribution, but package metadata doesn't have license text duplicated.